gradle.properties

```diff
@@ -14,7 +14,7 @@ gdxVersion=1.12.0
 
 # Publishing settings
 group=com.badlogicgames.gdx-video
-version=1.3.2
```

Comment:
I don't know if increasing the version is necessary. We haven't even made a full release yet. There are only snapshots.

Reply:
Ah, right. I'll remove that change for now.

Do you think a full release will happen at some point, or do you know what's missing for that?

Reply:
I can make a full release in the next few days. I didn't make it until now because I wanted the code to get more tested and the desktop implementation wasn't that good (like you experienced)